### PR TITLE
Update and rename checkov.yaml to kics.yaml

### PR DESCRIPTION
**Pull request by the-tatanka**

### CODE REVIEW

**Comment by bcronin90** [comment]
To clarify: I created a new branch off of this one and will create an identical PR, but with the file still named checkov.yaml. This is to allow for automatic testing. If those tests are successful, I will approve this PR and delete my own.

File: .github/workflows/kics.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: "KICS"
+
+on:
+  push:
+    branches: [main, master, develop]
+  # pull_request:
```
Comment (bcronin90) [inline .github/workflows/kics.yaml:6]: I'd replace these comments with:
```  pull_request:
    branches: [main, master, develop]```

Then the checks would run on PR creation.

**Comment by bcronin90** [comment]
Was this supposed to be merged in its current state?

**Comment by the-tatanka** [comment]
I mergerd both PRs - should be ok?

**Pull request merged**